presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -421,7 +421,7 @@ public void testDereferenceInFunctionCall()
     }
 
     @Test
-    public void testDereferenceInComparsion()
+    public void testDereferenceInComparison()
```

Comment:
this PR already depends on 3 other PRs, I do not want to add another dependency for such minor change